examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,45 @@
+import matplotlib.pyplot as plt
```

Comment:
Examples have a documentation string at the top to give this a title and a description of what an example  will cover. You can see the others files for examples.

Reply:
oh yeah, i am an idiot. How could i forget that. I will fix it.